slides/transactions.tex

```diff
@@ -0,0 +1,112 @@
+% Choose one to switch betweeen slides and handout
+\documentclass[]{beamer}
+%\documentclass[handout]{beamer}
+
+% Video Meta Data
+\title{Bitcoin, Blockchain and Cryptoassets}
+\subtitle{Transaction Overview}
+\author{Prof. Dr. Fabian Schär}
+\institute{University of Basel}
+
+% Config File
+\input{../config/config.tex}
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\begin{document}
+
+
+%%
+\thispagestyle{empty}
+\begin{frame}[noframenumbering]
+	\titlepage
+\end{frame}
+%%%
+
+
+%%%
+\begin{frame}{Structure of a Transaction}
+	\centering
+	\begin{figure}
+	\input{../assets/figures/transaction-structure.tex}
+	\end{figure} 
+\vspace{1em}
+\begin{itemize}
+  	\item<2->{Must contain at least one in- and output}
+  	\item<3->{Sum of inputs must be at least as high as sum of outputs}
+	\item<4->{Credit can be divided to any number of output}
```

Comment:
```suggestion
```

Reply:
Changed in commit 3b98723b34ab62fca4cbb8066335dd4f2d9224fa